File: 250324/배열의 특정 위치 찾기 2/find-specific-location-fo-array-2.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int arr[] = new int[10];
        int sum_even = 0, sum_odd = 0;
        for(int i = 0; i < 10; i++) {
            arr[i] = sc.nextInt();
            if(i % 2 == 0) {
                sum_odd += arr[i];
            } else {
                sum_even += arr[i];
            }
        }
        if(sum_odd >= sum_even) {
            System.out.println(sum_odd - sum_even);
        }
        else {
            System.out.println(sum_even - sum_odd);
        }
        sc.close();
    }
}

File: 250324/배열의 특정 위치 찾기 2/find-specific-location-fo-array-2.cpp
#include <iostream>
using namespace std;

int main()
{
    ios::sync_with_stdio(false);
    cin.tie(0);
    cout.tie(0);

    int sum_even = 0;
    int sum_odd = 0;
    int arr[10] = {0, };
    for(int i = 0; i < 10; i++) {
        cin >> arr[i];
        if(i % 2 == 0) {
            sum_odd += arr[i];
        }
        else {
            sum_even += arr[i];
        }
    }
    if(sum_odd >= sum_even) {
        cout << sum_odd - sum_even;
    }
    else {
        cout << sum_even - sum_odd;
    }
    return 0;
}